contracts/modules/LidoModule.sol

```diff
@@ -0,0 +1,58 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity 0.8.6;
+
+import '@yield-protocol/vault-interfaces/IJoin.sol';
+import '@yield-protocol/vault-interfaces/ICauldron.sol';
+import '@yield-protocol/utils-v2/contracts/token/IERC20.sol';
+import '@yield-protocol/utils-v2/contracts/token/TransferHelper.sol';
+import '@yield-protocol/utils-v2/contracts/math/WMul.sol';
+import '@yield-protocol/utils-v2/contracts/cast/CastU256U128.sol';
+import '@yield-protocol/utils-v2/contracts/cast/CastU256I128.sol';
+import '@yield-protocol/utils-v2/contracts/cast/CastU128I128.sol';
+import '../LadleStorage.sol';
+import '../oracles/lido/IWstETH.sol';
+
+/// @dev A ladle module to handle wrapping & unwrapping of stETH
+contract LidoModule is LadleStorage {
+    using WMul for uint256;
+    using CastU256U128 for uint256;
+    using CastU256I128 for uint256;
+    using CastU128I128 for uint128;
+    using TransferHelper for IERC20;
+    using TransferHelper for IWstETH;
+    using TransferHelper for IWETH9;
+    using TransferHelper for address payable;
+    IWstETH public wstETH;
+
+    constructor(
+        ICauldron cauldron,
+        IWETH9 steth,
```

Comment:
```suggestion
        IWETH9 weth,
```
I can't think of exactly how this would break. Maybe it wouldn't. However, it leads to unclear code.